CHANGES.md

```diff
@@ -17,6 +17,10 @@
 -   Export a few convenience types: `IAnySubFormAccessor`,
     `IAnyRepeatingFormAccessor`, `IAnyRepeatingFormIndexedAccessor`.
 
+-   Add a `maxZeroesPadding` option to the `decimal` renderer.
```

Comment:
`1.26.0` has already been released, you would need to make a new version to include this change :smile: